src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -193,7 +186,7 @@ export const ProjectAccessAssign = ({
                 await changeGroupRole(projectId, role.id, selected.entity.id);
             }
             refetchProjectAccess();
-            setOpen(false);
+            navigate(-1);
```

Comment:
Could we extract this readable constants? -1 is not very readable.

Reply:
Do you mean something like https://github.com/Unleash/unleash-frontend/pull/1178/commits/a1347b667f2b016d0a986dd1c73a9049c14b7712 ?